.changeset/beige-avocados-matter.md

```diff
@@ -0,0 +1,28 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Replace `createCatalogPolicyDecision` export with `createCatalogConditionalDecision`, which accepts a permission parameter of type `ResourcePermission<'catalog-entity'>` along with conditions. The permission passed is expected to be the handled permission in `PermissionPolicy#handle`, whose type must first be narrowed using methods like `isPermission` and `isResourcePermission`:
```

Comment:
Could you highlight that this is a breaking change of an alpha API? Both to signal the breakage but also to explain why it's done without a major bump

Reply:
Do we have a specific way to note breaking alpha apis? Perhaps `**BREAKING (alpha api):**`